src/Message/AcknowledgeResponse.php

```diff
@@ -9,6 +9,6 @@
      */
     public function isSuccessful()
     {
-        return true;
+        return parent::isSuccessful() && '204' === $this->request->getResponse()->getCode();
```

Comment:
Dit levert de foutmelding `You must call send() before accessing the Response` op

Je zou dan het volgende moeten aanpassen in `AcknowlegdeRequest::sendData`:
```
return $this->response = new AcknowledgeResponse(//...);
```

Dat gaat echter ook niet werken, aangezien `getCode` van de (Omnipay)ResponseInterface de `error_code` uit de content oplevert.

Je kunt dus beter een statusCode parameter en getter intoduceren op de response, en dan de request als volgt aanpassen:
```
$response = $this->sendRequest(RequestInterface::POST, $url, $data);

        return $this->response = new AcknowledgeResponse(
            $this,
            $this->getResponseBody($response),
            $response->getStatusCode()
        );
```